packages/components/src/custom-select-control-v2/legacy-component/index.tsx

```diff
@@ -60,56 +85,56 @@ function CustomSelectControl( props: LegacyCustomSelectProps ) {
 		defaultValue: options[ 0 ]?.name,
 	} );
 
-	const children = options.map(
```

Comment:
The following block of code is just about changing from  mostly `__experimentalHint` to `hint`, the rest is whitespace-related changes (indentation)